### PR TITLE
[WIP]Add basic evaluation code

### PR DESCRIPTION
**Pull request by sara-02**

### CODE REVIEW

File: src/evaluation/evaluation_metric.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        self.time = time.gmtime()
+
+    def generate_uid(self):
+        """Generate a unique time string for storing result."""
+        uid = ""
+        fields = self.time.n_sequence_fields
+        for i in range(fields):
+            uid = uid + "_" + str(self.time[i])
+        return uid + ".json"
+
```
Comment (tisnik) [inline src/evaluation/evaluation_metric.py]: I'd suggest to use time.strftime() there

**Comment by tisnik** [comment]
@sara-02 can you please resolve conflicts so we will be able to merge your PR? Thank you in advance.

**Comment by tisnik** [comment]
@sara-02 hi, is this PR still relevant? If not, please close it

**Comment by rootAvish** [comment]
@tisnik Even if she fixes it I think the PR depends on #30 to be merged first.

**Comment by rootAvish** [comment]
@sara-02 On the other hand, I think this can go in without the training code. Can you please get this merged today?

**Comment by sara-02** [comment]
@tisnik, I am marking this as WIP.

**Comment by rootAvish** [comment]
@sara-02 Can you address the bug we had talked about the other day? You said there are uncommitted changes on your local machine. I need this today to evaluate the couple of models that I've trained.

**Comment by sara-02** [comment]
Added. As for Recall@K, I have not added a separate function to save the recall yet.

**Comment by sara-02** [comment]
[test]

**Comment by rootAvish** [comment]
There's an implementation of this that lives as a part of the re-training pipeline. I'm closing this too.

**Pull request closed without merging**